scripts/local-universe.py

```diff
@@ -45,9 +45,9 @@ def main():
     parser.add_argument(
         '--include',
         default='',
-        help='Command separated list of packages to include. If this option '
-        'is not specified then all packages are downloaded. E.g. '
-        '--include="marathon,chronos"')
+        help='Command (,) separated list of packages to include. For each '
+        'package specify both the name and version by separating them with a '
+        'colon (:). E.g. --include="marathon:1.4.2,chronos:2.5.0"')
```

Comment:
Yes. Indirectly, users have told us that they want to be able to produce their Local Universe and in the process possibly add new packages. The best way to do that is to require that packages are enumerated explicitly.